dockerfiles/theia/Dockerfile

```diff
@@ -64,12 +64,10 @@ ARG CDN_PREFIX=""
 ARG MONACO_CDN_PREFIX=""
 WORKDIR ${HOME}/theia-source-code
 
-COPY che-theia/che-theia-init-sources.yml ${HOME}/che-theia-init-sources.yml
+COPY che-theia/che-theia.tar.gz ${HOME}/che-theia.tar.gz
+RUN mkdir ${HOME}/theia-source-code/che-theia && tar -xf ${HOME}/che-theia.tar.gz -C ${HOME}/theia-source-code/che-theia
```

Comment:
`ADD` instruction is able to unpack automatically if we give a .tgz no need to run `COPY` and then `RUN` tar

Reply:
Done